grasa_event_locator/settings/production.py

```diff
@@ -7,12 +7,13 @@
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.',
-        'NAME': '',
-        'USER': '',
-        'PASSWORD': '',
-        'HOST': '',
-        'PORT': '',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'grasa_event_locator',
+        'USER': 'grasaadmin',
+        'PASSWORD': 'djangoGrasa2019',
+        'HOST': 'db',
+        'PORT': '3306',
```

Comment:
These options should not be hard-coded into the configuration file. These values should be retrieved from environment variables since they are set in the `docker-compose` file (and in future production, they will be set in that environment).

Reply:
please see #12

Reply:
Same feedback as [here](https://github.com/jwflory/django-rit-grasa/pull/11/files#r326924238). Could you please configure this as the development database instead in `development.py`?